main.nf

```diff
@@ -860,6 +849,8 @@ if (!params.skip_alignment) {
     ch_sortbam_quant
         .map { it -> [ it[0], it[3] ] }
         .set { ch_sortbam_quant }
+    ch_sample_map_deseq2 = "F"
+    ch_sample_map_dexseq = "F"
```

Comment:
```suggestion
```

Reply:
What are these extra variables doing? The information in the processes I wrote should be enough to do a differential analysis. Just extract the group info directly from the counts matrix within the R script.

Reply:
You should just need to change the R scripts and uncomment out the code here to test. No need to change any of the logic.

Reply:
Hi @drpatelh, with the adjusted Rscript without the samplesheet parameter, the DESeq2 and the DEXSeq processes ran fine with fastq inputs but not bam inputs. For the fastq inputs, the sorted.bam file names inputted into bambu and stringtie2+featurecounts are renamed to Group_Replicate (ex. MCF7_R1), but for the bam inputs, the bam file names are not renamed, and bambu and stringtie2+featureCounts are using the full name of the bam files. The column names of count outputs from bambu/stringtie2+featureCounts from fastq and bam inputs are different: fastq inputs have column names in the format of Group_Replicate (ex.MCF7_R1), whereas bam inputs are using the full names of the input files, which can be anything, as column names.